From 2a105768b4192dce47076a6cafaab82c8974222e Mon Sep 17 00:00:00 2001
From: Mandy Lavi <mandy.lavi@freescale.com>
Date: Mon, 19 May 2014 19:09:39 +0300
Subject: [PATCH 350/466] fmd: change p1023 DEFAULT_totalFifoSize to 49KB

49KB can accomodate buffers for 2x1G + 1xOH + 1xOP

Change-Id: Ib2da4bb8e9d393f207bb756cf1b5a3ab1e75cdd2
Signed-off-by: Mandy Lavi <mandy.lavi@freescale.com>
Reviewed-on: http://git.am.freescale.net:8181/12499
Reviewed-by: Nir Erez <nir.erez@freescale.com>
Reviewed-by: Jose Rivera <German.Rivera@freescale.com>
Tested-by: Jose Rivera <German.Rivera@freescale.com>
[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 drivers/net/ethernet/freescale/fman/Peripherals/FM/fm.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/fman/Peripherals/FM/fm.h b/drivers/net/ethernet/freescale/fman/Peripherals/FM/fm.h
index 59a78e9..07431c3 100644
--- a/drivers/net/ethernet/freescale/fman/Peripherals/FM/fm.h
+++ b/drivers/net/ethernet/freescale/fman/Peripherals/FM/fm.h
@@ -343,7 +343,7 @@ switch (exception){                                         \
 #define DEFAULT_totalFifoSize(major, minor)     \
     (((major == 2) || (major == 5)) ?           \
      (100*KILOBYTE) : ((major == 4) ?           \
-     (46*KILOBYTE) : (122*KILOBYTE)))
+     (49*KILOBYTE) : (122*KILOBYTE)))
 #define DEFAULT_totalNumOfTasks             BMI_MAX_NUM_OF_TASKS
 
 #define DEFAULT_dmaCommQLow                 ((DMA_THRESH_MAX_COMMQ+1)/2)
-- 
1.7.10.4

